test/test.js

```diff
@@ -50,24 +50,24 @@ More commands and details:
 		}).catch(e => done(e));
 	});
 
-	it("good evening", function(done) {
-		const helper = new Helper('../scripts/polite.js');
+	it("blahwithspace", function(done) {
+		const helper = new Helper('../scripts/search.js');
 		room = helper.createRoom();
 		const command = '@hubot ' + this.test.title;
 		co(function *() {
 			yield room.user.say('alice', command);
-			expect(room.messages.length).to.eql(2);
+			setTimeout(() => expect(room.messages.length).to.eql(2), 100);
 			done();
 		}).catch(e => done(e));
 	});
 
-	it("recommend apple", function(done) {
-		const helper = new Helper('../scripts/recommend.js');
+	it("blahnospace", function(done) {
+		const helper = new Helper('../scripts/search.js');
 		room = helper.createRoom();
-		const command = '@hubot ' + this.test.title;
+		const command = '@hubot' + this.test.title;
 		co(function *() {
 			yield room.user.say('alice', command);
-			expect(room.messages.length).to.eql(2);
+			setTimeout(() => expect(room.messages.length).to.eql(2), 100);
```

Comment:
The same applies here about testing the message contents and placing `done()` inside the timeout callback.